### DISCUSSION TITLE
AWS Lambda, change the priority so the response transfer is useful with it.

### DISCUSSION THREAD

**Discussion by mickeyr** [Kong/kong#13308] [Ideas and feature requests]
The `response-transformer` plugin currently has a priority of 800 and the `aws-lambda` plugin has a priority of 750.  Because of this, I am unable to configure the `response-transformer` to remove headers from responses when using the the `aws-lambda` plugin. 

The use case is that I'd like to remove the `x-amzn-*` headers from the response.

**Comment by hanshuebner** [comment]
The priority of an existing plugin cannot be changed, but you may be able to use the [post-function plugin](https://docs.konghq.com/hub/kong-inc/post-function/) to modify the response header according to your needs.

**Reply by chobits** [reply]
I think its a good idea, do you think it is ok? @mickeyr

**Reply by mickeyr** [reply]
Yeah, this solves the issue, just not as elegant I suppose

**Reply by chobits** [reply]
I understand that some basic plugins like response-transformer should have higher priority to have full power to modify the response without interference from other plugins. However, in reality, modifying the priority might significantly impact current users.

**Reply by chobits** [reply]
But this is a very meaningful discussion, thank you for bringing it up
